infrastructure/ArmTemplates/antareskvtemplate.json

```diff
@@ -352,7 +352,29 @@
                 "Name": "WEBSITE_LOCAL_CACHE_OPTION",
                 "Value": "Always"
               }
-            ]
+            ],
+            "autoHealEnabled": true,
+            "autoHealRules": {
+              "triggers": {
+                "requests": null,
+                "privateBytesInKB": 0,
+                "statusCodes": [
+                  {
+                      "status": 500,
+                      "subStatus": 0,
+                      "win32Status": 0,
+                      "count": 100,
+                      "timeInterval": "00:10:00"
+                  }
+                ],
+                "slowRequests": null
+              },
+              "actions": {
+                "actionType": "Recycle",
```

Comment:
can we collect dumps and then recycle

Reply:
Also is there a way to recycle for StatusCode >=500

Reply:
there is no way to specify status code above 500 we can specify multiple unique status codes though. are there any others you can think of?

Reply:
503 would be good to add.
The substatus and win32 is optional I believe as per Autoheal UI, so we can prob ignore those